### PR TITLE
first UI for Interprep

### PR DESCRIPTION
**Pull request by sanggwonjang**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,173 +1,493 @@
...
-      child: SizedBox(
-        width: 400,
-        child: Card(
-          child: SignUpForm(),
+        appBar: AppBar(
+          title: Text("Interprep"),
+          centerTitle: true,
+          backgroundColor: Colors.brown[300],
+          actions: <Widget>[
+            IconButton(
```
Comment (hyeonwooshim) [inline lib/main.dart:18]: What is this button for?

File: lib/main.dart [hunk 2]
```diff
@@ -1,173 +1,493 @@
...
+  "Jude",
+  "Revelation",
+];
 
-  @override
-  void initState() {
-    super.initState();
-  }
+enum verseStatus { recited, read }
+enum verseLocation { before, after }
```
Comment (hyeonwooshim) [inline lib/main.dart:175]: First letter should be capitalized.

File: lib/main.dart (hunk 1 above)
Comment (sanggwonjang) [reply lib/main.dart:18]: Its for the look :)

Comment (hyeonwooshim) [reply lib/main.dart:18]: Why not pass in null for `onPressed` if the button isn't supposed to do anything?

File: lib/main.dart [hunk 3]
```diff
@@ -1,173 +1,378 @@
...
 
-class SignUpFormBody extends StatefulWidget {
-  final ValueChanged<double> onProgressChanged;
+KoreanBible koreanBible = new KoreanBible();
+NkjvBible nkjvBible = new NkjvBible();
 
-  SignUpFormBody({
-    @required this.onProgressChanged,
-  }) : assert(onProgressChanged != null);
+List<String> suggestions = koreanBible.bookNames + nkjvBible.bookNames;
```
Comment (hyeonwooshim) [inline lib/main.dart:44]: Hmm You don't need to instantiate KoreanBible and NkjvBible here.
You can access `KoreanBible.fullBookNames` and `NkjvBible.fullBookNames`, the static versions of the list.

File: lib/main.dart [hunk 4]
```diff
@@ -1,173 +1,378 @@
...
+                  //   child: Text('Book Name :',
+                  //       textAlign: TextAlign.right,
+                  //       style: TextStyle(
+                  //         fontSize: 15,
+                  //         fontWeight: FontWeight.bold,
+                  //       )),
+                  //   alignment: Alignment.centerRight,
+                  //   width: (MediaQuery.of(context).size.width / 2.5) / 3.5,
+                  //   height: (MediaQuery.of(context).size.height / 1.25) / 17,
+                  // ),
```
Comment (hyeonwooshim) [inline lib/main.dart:213]: Could you remove these commented-out lines?

**Pull request merged**